app/main/posts/views/post-view-map.html

```diff
@@ -3,7 +3,7 @@
     <div ui-view="mode-context" noanimation="true" filters="filters"></div>
     <main role="main">
 		<post-toolbar is-loading="isLoading" filters="filters" edit-mode="{ editing: false }"></post-toolbar>
-    	<div class="map-view">
+    	<div class="map-view" id="map-view">
 		    <div id="map-full-size" class="map full-size" ng-show="!noui"></div>
```

Comment:
I don't think the id is needed here, we use the ids in the child-divs instead, one for using the normal view you see when navigating to the map-view and one for specific use cases when no ui around the map is needed.